document/core/binary/instructions.rst

```diff
@@ -442,34 +442,37 @@ The |VCONST| instruction is followed by 16 immediate bytes, which are converted
    \end{array}
 
 .. _binary-vternop:
+.. _binary-laneidx:
 
-The |SHUFFLE| instruction is also followed by 16 immediate bytes:
+The |SHUFFLE| instruction is also followed by the encoding of 16 |laneidx| immediates.
 
 .. math::
    \begin{array}{llclll}
+   \production{lane index} & \Blaneidx &::=&
+     l{:}\Bu8 &\Rightarrow& l \\
```

Comment:
Note that Bu8 actually is a LEB128 representation. Here, you'll still need to use Bbyte.

Reply:
Good catch, fixed in #357